api/test/integration/test_security_DELETE_endpoints.tavern.yaml

```diff
@@ -577,7 +571,7 @@ test_name: CLEANER DELETE /security/{roles}
 
 stages:
 
-  # DELETE /security/roles
+  # DELETE /security/roles, this tests should fail in white mode
```

Comment:
I don´t understand this. Why should they fail, these tests are supposed to be run only in black_mode or in white_mode for admin user.

Reply:
Because before this test we delete all policies and the RBAC mode is white so all of the following requests will be rejected.

Reply:
That' s true with the last update this test can be passed in both modes, policies and administrator roles are used.